File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/AuthOnlyResponse.java
package com.payhub.ws.model;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL) 
public class AuthOnlyResponse {
	
	        private String transactionId;
	        
	        private String approvalCode;
	        
	        private String processedDateTime;
	        
	        private String avsResultCode;
	        
	        private String verificationResultCode;
	        
	        private String batchId;
	        
	        private String responseCode;
	        
	        private String responseText;
	        
	        private String cisNote;
	        
	        private String riskStatusResponseText;
	        
	        private String riskStatusRespondeCode;
	        
	        private String dateTime;
	        
	        private String tokenizedCard;
	        
	        private BillingReferences billingReferences;
	        
	        private CustomerReference customerReference;

			public String getTransactionId() {
				return transactionId;
			}

			public void setTransactionId(String transactionId) {
				this.transactionId = transactionId;
			}

			public String getApprovalCode() {
				return approvalCode;
			}

			public void setApprovalCode(String approvalCode) {
				this.approvalCode = approvalCode;
			}

			public String getProcessedDateTime() {
				return processedDateTime;
			}

			public void setProcessedDateTime(String processedDateTime) {
				this.processedDateTime = processedDateTime;
			}

			public String getAvsResultCode() {
				return avsResultCode;
			}

			public void setAvsResultCode(String avsResultCode) {
				this.avsResultCode = avsResultCode;
			}

			public String getVerificationResultCode() {
				return verificationResultCode;
			}

			public void setVerificationResultCode(String verificationResultCode) {
				this.verificationResultCode = verificationResultCode;
			}

			public String getBatchId() {
				return batchId;
			}

			public void setBatchId(String batchId) {
				this.batchId = batchId;
			}

			public String getResponseCode() {
				return responseCode;
			}

			public void setResponseCode(String responseCode) {
				this.responseCode = responseCode;
			}

			public String getResponseText() {
				return responseText;
			}

			public void setResponseText(String responseText) {
				this.responseText = responseText;
			}

			public String getCisNote() {
				return cisNote;
			}

			public void setCisNote(String cisNote) {
				this.cisNote = cisNote;
			}

			public String getRiskStatusResponseText() {
				return riskStatusResponseText;
			}

			public void setRiskStatusResponseText(String riskStatusResponseText) {
				this.riskStatusResponseText = riskStatusResponseText;
			}

			public String getRiskStatusRespondeCode() {
				return riskStatusRespondeCode;
			}

			public void setRiskStatusRespondeCode(String riskStatusRespondeCode) {
				this.riskStatusRespondeCode = riskStatusRespondeCode;
			}

			public String getDateTime() {
				return dateTime;
			}

			public void setDateTime(String dateTime) {
				this.dateTime = dateTime;
			}

			public String getTokenizedCard() {
				return tokenizedCard;
			}

			public void setTokenizedCard(String tokenizedCard) {
				this.tokenizedCard = tokenizedCard;
			}

			public BillingReferences getBillingReferences() {
				return billingReferences;
			}

			public void setBillingReferences(BillingReferences billingReferences) {
				this.billingReferences = billingReferences;
			}

			public CustomerReference getCustomerReference() {
				return customerReference;
			}

			public void setCustomerReference(CustomerReference customerReference) {
				this.customerReference = customerReference;
			}
	        
	        
}

File: php_SDK/com/payhub/ws/model/AuthOnlyResponse.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 23/07/2015
 * Time: 12:28
 */
class AuthOnlyResponse
{
    private $transactionId;

    private $approvalCode;

    private $processedDateTime;

    private $avsResultCode;

    private $verificationResultCode;

    private $batchId;

    private $responseCode;

    private $responseText;

    private $cisNote;

    private $riskStatusResponseText;

    private $riskStatusRespondeCode;

    private $dateTime;

    private $tokenizedCard;

    private  $billingReferences;

    private  $customerReference;

    /**
     * @return mixed
     */
    public function getTransactionId()
    {
        return $this->transactionId;
    }

    /**
     * @param mixed $transactionId
     */
    public function setTransactionId($transactionId)
    {
        $this->transactionId = $transactionId;
    }

    /**
     * @return mixed
     */
    public function getApprovalCode()
    {
        return $this->approvalCode;
    }

    /**
     * @param mixed $approvalCode
     */
    public function setApprovalCode($approvalCode)
    {
        $this->approvalCode = $approvalCode;
    }

    /**
     * @return mixed
     */
    public function getProcessedDateTime()
    {
        return $this->processedDateTime;
    }

    /**
     * @param mixed $processedDateTime
     */
    public function setProcessedDateTime($processedDateTime)
    {
        $this->processedDateTime = $processedDateTime;
    }

    /**
     * @return mixed
     */
    public function getAvsResultCode()
    {
        return $this->avsResultCode;
    }

    /**
     * @param mixed $avsResultCode
     */
    public function setAvsResultCode($avsResultCode)
    {
        $this->avsResultCode = $avsResultCode;
    }

    /**
     * @return mixed
     */
    public function getVerificationResultCode()
    {
        return $this->verificationResultCode;
    }

    /**
     * @param mixed $verificationResultCode
     */
    public function setVerificationResultCode($verificationResultCode)
    {
        $this->verificationResultCode = $verificationResultCode;
    }

    /**
     * @return mixed
     */
    public function getBatchId()
    {
        return $this->batchId;
    }

    /**
     * @param mixed $batchId
     */
    public function setBatchId($batchId)
    {
        $this->batchId = $batchId;
    }

    /**
     * @return mixed
     */
    public function getResponseCode()
    {
        return $this->responseCode;
    }

    /**
     * @param mixed $responseCode
     */
    public function setResponseCode($responseCode)
    {
        $this->responseCode = $responseCode;
    }

    /**
     * @return mixed
     */
    public function getResponseText()
    {
        return $this->responseText;
    }

    /**
     * @param mixed $responseText
     */
    public function setResponseText($responseText)
    {
        $this->responseText = $responseText;
    }

    /**
     * @return mixed
     */
    public function getCisNote()
    {
        return $this->cisNote;
    }

    /**
     * @param mixed $cisNote
     */
    public function setCisNote($cisNote)
    {
        $this->cisNote = $cisNote;
    }

    /**
     * @return mixed
     */
    public function getRiskStatusResponseText()
    {
        return $this->riskStatusResponseText;
    }

    /**
     * @param mixed $riskStatusResponseText
     */
    public function setRiskStatusResponseText($riskStatusResponseText)
    {
        $this->riskStatusResponseText = $riskStatusResponseText;
    }

    /**
     * @return mixed
     */
    public function getRiskStatusRespondeCode()
    {
        return $this->riskStatusRespondeCode;
    }

    /**
     * @param mixed $riskStatusRespondeCode
     */
    public function setRiskStatusRespondeCode($riskStatusRespondeCode)
    {
        $this->riskStatusRespondeCode = $riskStatusRespondeCode;
    }

    /**
     * @return mixed
     */
    public function getDateTime()
    {
        return $this->dateTime;
    }

    /**
     * @param mixed $dateTime
     */
    public function setDateTime($dateTime)
    {
        $this->dateTime = $dateTime;
    }

    /**
     * @return mixed
     */
    public function getTokenizedCard()
    {
        return $this->tokenizedCard;
    }

    /**
     * @param mixed $tokenizedCard
     */
    public function setTokenizedCard($tokenizedCard)
    {
        $this->tokenizedCard = $tokenizedCard;
    }

    /**
     * @return mixed
     */
    public function getBillingReferences()
    {
        return $this->billingReferences;
    }

    /**
     * @param mixed $billingReferences
     */
    public function setBillingReferences($billingReferences)
    {
        $this->billingReferences = $billingReferences;
    }

    /**
     * @return mixed
     */
    public function getCustomerReference()
    {
        return $this->customerReference;
    }

    /**
     * @param mixed $customerReference
     */
    public function setCustomerReference($customerReference)
    {
        $this->customerReference = $customerReference;
    }
    public static function fromArray($data){
        if(!is_null($data)) {
            $auth = new AuthOnlyResponse();
            foreach ($data as $key => $value) {
                if ($key == "billingReferences") {
                    $auth->{$key} = BillingReferences::fromArray($value);
                } elseif ($key == "customerReference") {
                    $auth->{$key} = CustomerReference::fromArray($value);
                } else {
                    $auth->{$key} = $value;
                }
            }
            return $auth;
        }
    }

}